### ISSUE TITLE
Transcoding and playback, timeline not showing max video length

### ISSUE THREAD

**Issue by SablyTv** [jellyfin/jellyfin#10569]
**Media Info of the file**
General
Unique ID                                : 277878905945664979951874495736790725561 (0xD10D87D493F4527618906938B75D33B9)
Complete name                            : \\192.168.0.190\tank\Media\Movies\Pulse (2001)\Pulse (Kairo) 2001 BD 720p (x265).mkv
Format                                   : Matroska
Format version                           : Version 4
File size                                : 875 MiB
Duration                                 : 1 h 58 min
Overall bit rate                         : 1 028 kb/s
Frame rate                               : 23.976 FPS
Writing application                      : Lavf58.10.100
Writing library                          : Lavf58.10.100
ErrorDetectionType                       : Per level 1

Video
ID                                       : 1
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main 10@L3.1@Main
Codec ID                                 : V_MPEGH/ISO/HEVC
Duration                                 : 1 h 58 min
Bit rate                                 : 13.0 Mb/s
Width                                    : 1 280 pixels
Height                                   : 720 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 23.976 (23976/1000) FPS
Original frame rate                      : 23.976 (24000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 10 bits
Bits/(Pixel*Frame)                       : 0.590
Stream size                              : 10.8 GiB
Writing library                          : x265 2.7+8-613d9f443769:[Windows][GCC 7.3.0][64 bit] 10bit
Encoding settings                        : cpuid=1173503 / frame-threads=2 / numa-pools=4 / wpp / no-pmode / no-pme / no-psnr / no-ssim / log-level=2 / input-csp=1 / input-res=1280x720 / interlace=0 / total-frames=0 / level-idc=0 / high-tier=1 / uhd-bd=0 / ref=3 / no-allow-non-conformance / no-repeat-headers / annexb / no-aud / no-hrd / info / hash=0 / no-temporal-layers / open-gop / min-keyint=23 / keyint=250 / gop-lookahead=0 / bframes=4 / b-adapt=2 / b-pyramid / bframe-bias=0 / rc-lookahead=20 / lookahead-slices=4 / scenecut=40 / radl=0 / no-intra-refresh / ctu=64 / min-cu-size=8 / no-rect / no-amp / max-tu-size=32 / tu-inter-depth=1 / tu-intra-depth=1 / limit-tu=0 / rdoq-level=0 / dynamic-rd=0.00 / no-ssim-rd / signhide / no-tskip / nr-intra=0 / nr-inter=0 / no-constrained-intra / strong-intra-smoothing / max-merge=2 / limit-refs=3 / no-limit-modes / me=1 / subme=2 / merange=57 / temporal-mvp / weightp / no-weightb / no-analyze-src-pics / deblock=0:0 / sao / no-sao-non-deblock / rd=3 / no-early-skip / rskip / no-fast-intra / no-tskip-fast / no-cu-lossless / no-b-intra / no-splitrd-skip / rdpenalty=0 / psy-rd=2.00 / psy-rdoq=0.00 / no-rd-refine / no-lossless / cbqpoffs=0 / crqpoffs=0 / rc=crf / crf=23.0 / qcomp=0.60 / qpstep=4 / stats-write=0 / stats-read=0 / ipratio=1.40 / pbratio=1.30 / aq-mode=1 / aq-strength=1.00 / cutree / zone-count=0 / no-strict-cbr / qg-size=32 / no-rc-grain / qpmax=69 / qpmin=0 / no-const-vbv / sar=1 / overscan=0 / videoformat=5 / range=0 / colorprim=2 / transfer=2 / colormatrix=2 / chromaloc=0 / display-window=0 / max-cll=0,0 / min-luma=0 / max-luma=1023 / log2-max-poc-lsb=8 / vui-timing-info / vui-hrd-info / slices=1 / no-opt-qp-pps / no-opt-ref-list-length-pps / no-multi-pass-opt-rps / scenecut-bias=0.05 / no-opt-cu-delta-qp / no-aq-motion / no-hdr / no-hdr-opt / no-dhdr10-opt / analysis-reuse-level=5 / scale-factor=0 / refine-intra=0 / refine-inter=0 / refine-mv=0 / no-limit-sao / ctu-info=0 / no-lowpass-dct / refine-mv-type=0 / copy-pic=1 / max-ausize-factor=1.0
Language                                 : English
Default                                  : Yes
Forced                                   : No

Audio
ID                                       : 2
Format                                   : Opus
Codec ID                                 : A_OPUS
Duration                                 : 1 h 58 min
Bit rate                                 : 1 423 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 11.719 FPS (4096 SPF)
Bit depth                                : 24 bits
Compression mode                         : Lossy
Stream size                              : 1.18 GiB
Writing library                          : Lavc58.14.100 libopus
Language                                 : Japanese
Default                                  : Yes
Forced                                   : No

Text
ID                                       : 3
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Duration                                 : 1 h 58 min
Bit rate                                 : 19 b/s
Frame rate                               : 0.111 FPS
Count of elements                        : 789
Stream size                              : 17.0 KiB (0%)
Title                                    : forced
Language                                 : English
Default                                  : Yes
Forced                                   : Yes

Menu
00:00:00.000                             : 00:00:00.000
00:08:38.935                             : 00:08:38.935
00:18:16.220                             : 00:18:16.220
00:31:18.835                             : 00:31:18.835
00:39:22.193                             : 00:39:22.193
00:49:28.632                             : 00:49:28.632
00:59:26.313                             : 00:59:26.313
01:12:02.068                             : 01:12:02.068
01:20:56.727                             : 01:20:56.727
01:29:42.377                             : 01:29:42.377
01:39:05.815                             : 01:39:05.815
01:46:36.599                             : 01:46:36.599

**Question**
I basically just have a question. When Jellyfin starts a movie we can see that the "Max video length" is not set as the actual length of the video. It basically shows us what we have transcoded so far (image below). Googled and I think this is because of HLS. Even though we are transcoding I would like to always show the maximum length of the video. How could I do this?

It worked as I wanted when running everything in docker. But when I switched to service using TrueNAS in a jail this is what I get. The "Maximum length" obviously gets longer and longer while we keep transcoding.

![image](https://github.com/jellyfin/jellyfin/assets/44623142/d22b5163-1175-4259-8144-37b429b397be)

**Comment by felix920506** [comment]
What client are you using? Never happened to me on JMP or Web

**Comment by hthetiot** [comment]
Had the same issue via DLNA on Samsung TV client.

**Comment by sevenrats** [comment]
@SablyTv can you please confirm that you were using DLNA when you saw this issue?

**Comment by hthetiot** [comment]
Note: I found that the issue occur when using DLNA and H265 file, when media is H264 no skeek and timeline issues.

**Comment by SablyTv** [comment]
> @SablyTv can you please confirm that you were using DLNA when you saw this issue?

Hey sorry for late reply. Im not too sure. I fixed it by reinstalling everything. I was on Truenas Core before where the issue appered. Now im on Truenas Scale instead which works great.

**Comment by fazadaoc** [comment]
I am very sorry for necro-ing this issue but it is the only hit on google. I am having a similar issue where transcoded media is not showing the full timeline but (i assume) the length of the available transcode stream. This happens on 4k hevc hdr media. I am using jellyfin browser web payer

<img width="3860" height="1778" alt="Image" src="https://github.com/user-attachments/assets/bf548598-8c9f-4efa-aafc-3d1bdbe976a3" />

**Comment by literallylara** [comment]
~~For me it was caused by having my library folder path being a mapping from `fuse.bindfs`, meaning the files were not accessed directly but through the fuse layer. Wasn't able to find the exact cause yet though.~~

Update: It was actually caused by `jellyfin-ffmpeg` (v7.1.1). When I moved away from bindfs I continued testing on a client that did not need encoding, causing me to believe that bindfs was the issue. Upon testing on a different client without bindfs I encountered the issue again. Once I reverted to the system-provided ffmpeg the issue disappeared and transcoding works again.

**ffmpeg:** 7.1.1-Jellyfin
**jellyfin:** 10.10.7.0
**filesystem:** zfs-2.3.3-1
**kernel:** 6.15.4-200.fc42.x86_64
**distribution:** Fedora Server 42
